docs/content/docs/changelogs/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Working changelog
+draft: true
+description: Explanations of breaking and notable changes go here.  Images should live in docs/static/images.  The contents of this file are moved to a release notes page as parts of building docs for each release.
+---
+
+## 0.48 Breaking and notable changes
+This release is the Public Preview!
```

Comment:
```suggestion
## 0.48 Breaking and notable changes

This release is the Public Preview!
```

Nit: markdown is line-break sensitive, so technically without the extra line break it's all just one long header line. I think most markdown parsers handle it correctly, but I am a pedant. :)

Reply:
Not sure the "public preview" designation is needed. Something might change, and I don't think we're using that terminology anywhere else.

Reply:
will remove the line.  wasn't aware of the spec on line breaks, good to know.  i've definitely just been doing the look-and-see approach with the parsers🤐

Reply:
```suggestion
## 0.48 Breaking and notable changes
```